MatrixSDK/Crypto/Algorithms/Megolm/MXMegolmEncryption.m

```diff
@@ -166,6 +190,7 @@ - (BOOL)isSessionSharingHistory:(MXOutboundSessionInfo *)session
  @param failure A block object called when the operation fails.
 */
 - (MXHTTPOperation *)getDevicesInRoom:(NSArray<NSString*>*)users
```

Comment:
This method includes documentation above so would be good to include the new parameter. Also perhaps just a matter of personal preference, but the method `getDevicesInRoom` does not suggest anything about distributing keys so perhaps the new parameter would be clearer as `includeUnverifiedUsers` rather than `forceDistributeToUnverified`